CondCore/Utilities/python/cond2xml.py

```diff
@@ -143,7 +143,7 @@ def prepPayload2xml(self, payloadType):
         	 codeFile.write(code)
     	 	 codeFile.close()
     
-    	cmd = "source /afs/cern.ch/cms/cmsset_default.sh;"
+    	cmd = "source /cvmfs/cms.cern.ch/cmsset_default.sh;"
```

Comment:
Does this work if person is not using sh-compatible shell?
Why don't you use `$CMS_PATH/cmsset_default.{sh,csh}`?

Reply:
@davidlt It's a good suggestion for the $CMS_PATH. For the shell type, is this required, considering that the command will be executed by python?

Reply:
No, required. I looked again at documentation and only now noticed:
```
On Unix with shell=True, the shell defaults to /bin/sh.
```
So it's bash/dash/zsh/whatever-sh-compatible

Reply:
thanks, I'm preparing a new PR...